Mathlib/Order/Hom/WithTopBot.lean

```diff
@@ -161,11 +161,14 @@ namespace OrderIso
 variable [PartialOrder α] [PartialOrder β] [PartialOrder γ]
 
 /-- A version of `Equiv.optionCongr` for `WithTop`. -/
-@[simps! apply]
 def withTopCongr (e : α ≃o β) : WithTop α ≃o WithTop β :=
   { e.toOrderEmbedding.withTopMap with
     toEquiv := e.toEquiv.optionCongr }
```

Comment:
I'd argue we should redefine this so that `simps` does do the right thing; I think in practice that would look like
```suggestion
def withTopCongr (e : α ≃o β) : WithTop α ≃o WithTop β where
  __ := e.toOrderEmbedding.withTopMap
  __ := e.toEquiv.optionCongr
```